app/src/main/java/com/example/thecocktailsapp/viewmodule/CocktailsAppViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.thecocktailsapp.viewmodule
+
+import android.content.ContentValues
+import android.util.Log
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.example.thecocktailsapp.model.CocktailIInfo.CocktailsResponse
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+
+class CocktailsAppViewModel(
+    private val cocktailsRepository: CocktailsRepository
+): ViewModel() {
+
+    // Expose screen UI state
+    private val ioDispatcher: CoroutineDispatcher = Dispatchers.IO
```

Comment:
Lets have this in the constructor, in order to pass a test dispatcher when testing